lib/resources/r4/charge_item.yaml

```diff
@@ -100,25 +100,51 @@ fields:
     description: A defined group of bill codes that drives billing behavior.
     url: https://fhir.cerner.com/millennium/r4/financial/general/charge-item#extensions
     binding:
-      description: A defined group of bill codes that drives billing behavior. This binding is used in <extension>bill-code-schedule</extension> extensions.
+      description: A defined group of bill codes that drives billing behavior. This binding is used in <code>bill-code-schedule</code> extensions.
       terminology:
         - display: Bill Code Schedule
           system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/14002
           info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-14002-bill-code-schedule
 
+  - name: Unit Price/Net Price
+    terminology_name: .extension[x].valueMoney.currency
```

Comment:
.extension?

Reply:
it will be like this Chargeitem..extension[x].valueMoney.currency

Reply:
Done https://github.com/cerner/fhir.cerner.com/pull/727/commits/82ebf2e77d51e32a3e17f1d414b1b8f8d414821f#diff-7ab995fcc2135db120dc94b545fa48898edd21d9d220eee5efae95920c369dcbR110